tests/hipify-clang/headers_test_03.cu

```diff
@@ -0,0 +1,10 @@
+// RUN: %run_test hipify "%s" "%t" %cuda_args
+
+// CHECK: #pragma once
+// CHECK-NEXT: #include <hip/hip_runtime.h>
+#pragma once
+// CHECK-NOT: #include<hip/hip_runtime.h>
```

Comment:
should it be a space after `#include`? possible it should be FILECHECK regex for a case hard-coded string would be changed?

Reply:
It doesn't matter, as FileCheck by default canonicalizes input horizontal whitespace (spaces and tabs). It matters only with --strict-whitespace option, which we do not set.